src/NzbDrone.Common/Instrumentation/NzbDroneLogger.cs

```diff
@@ -97,10 +99,21 @@ private static void RegisterDebugger()
             target.Layout = "[${level}] [${threadid}] ${logger}: ${message} ${onexception:inner=${newline}${newline}[v${assembly-version}] ${exception:format=ToString}${newline}${exception:format=Data}${newline}}";
 
             var loggingRule = new LoggingRule("*", LogLevel.Trace, target);
+
             LogManager.Configuration.AddTarget("debugger", target);
             LogManager.Configuration.LoggingRules.Add(loggingRule);
         }
 
+        private static void RegisterGlobalFilters()
+        {
+            LogManager.Setup().LoadConfiguration(c =>
+            {
+                c.ForLogger("Microsoft.Hosting.Lifetime*").WriteToNil(LogLevel.Warn);
```

Comment:
Can get rid of this one given we are not logging it at a separate level than `Microsoft*`

Reply:
Believe `Microsoft.Hosting.Lifetime*` should have finalMinLevel=`LogLevel.Info` (instead of `LogLevel.Warn`)

```csharp
                c.ForLogger("System.*").WriteToNil(LogLevel.Warn);
                c.ForLogger("Microsoft.*").WriteToNil(LogLevel.Warn);
                c.ForLogger("Microsoft.Hosting.Lifetime*").WriteToNil(LogLevel.Info);
```

Then it would match the Microsoft Extension LoggerFactory filters.